aws/accountid_endpoint_mode.go

```diff
@@ -0,0 +1,12 @@
+package aws
+
+// AccountIDEndpointMode switches on/off the account ID based endpoint routing
+type AccountIDEndpointMode string
+
+// enums of valid AccountIDEndpointMode
```

Comment:
Rather than having the comment on the declaration block, give each one a comment explaining what it does.